packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html

```diff
@@ -97,32 +101,36 @@
       </dxi-item>
 
       <dxi-item>
-        <form-item-date label="Start Date"
-                        text="Set Start Date"
-                        [isEditing]="isEditing"
-                        [(value)]="task.startDate"
+        <form-item-date
+          label="Start Date"
+          text="Set Start Date"
+          [isEditing]="isEditing"
+          [(value)]="task.startDate"
         ></form-item-date>
       </dxi-item>
 
       <dxi-item>
-        <form-item-date label="Due Date"
-                        text="Set Due Date"
-                        [isEditing]="isEditing"
-                        [(value)]="task.dueDate"
+        <form-item-date
+          label="Due Date"
+          text="Set Due Date"
+          [isEditing]="isEditing"
+          [(value)]="task.dueDate"
         ></form-item-date>
       </dxi-item>
-
     </dxi-item>
 
     <dxi-item [colSpan]="2">
-      <edit-view-item *ngIf="!isEditing"
-                      [isEditing]="isEditing"
-                      label="Details"
-                      [value]="task.description"
+      <edit-view-item
```

Comment:
for me, this less readable than previous variant, and it increase rows in file

Reply:
It's about linter config.